redwood-lazylayout-compose/src/commonMain/kotlin/app/cash/redwood/lazylayout/compose/modifier.kt

```diff
@@ -0,0 +1,19 @@
+package app.cash.redwood.lazylayout.compose
+
+import app.cash.redwood.layout.api.CrossAxisAlignment
+import app.cash.redwood.layout.modifier.Alignment
+
+internal class AlignmentImpl(
```

Comment:
not a data class?

Reply:
It's copied generated code from `redwood-layout-compose`. Ultimately I want to share `modifier` definitions with `redwood-layout`, so this is somewhat of a stop gap in the meantime.